tests/test_annotated.py

```diff
@@ -22,22 +22,22 @@
         (
             lambda: Annotated[int, Field(gt=0)],
             5,
-            'FieldInfo(annotation=int, required=False, default=5, metadata=[Gt(gt=0)])',
+            'FieldInfo(annotation=int, required=False, default=5, metadata=[Gt(gt=0)], frozen=False)',
```

Comment:
if we set the default of frozen to `False`, we should ignore `frozen=False` in repr.

Reply:
ignored